projects/1.18/assets/untamedwilds/untamedwilds/lang/zh_cn.json

```
@@ -0,0 +1,726 @@
+  "entity.untamedwilds.giant_salamander_hellbender": "美洲大鲵",
+  "entity.untamedwilds.giant_salamander_hellbender.sciname": "Cryptobranchus alleganiensis",
+  "item.untamedwilds.bucket_giant_salamander_chinese": "桶装中国大鲵",
+  "item.untamedwilds.bucket_giant_salamander_hellbender": "桶装日本大鲵",
+  "item.untamedwilds.bucket_giant_salamander_japanese": "桶装美洲大鲵",
+  
+  "item.untamedwilds.newt_spawn_egg": "蝾螈刷怪蛋",
+  "item.untamedwilds.egg_newt": "蝾螈蛋",
+  "entity.untamedwilds.newt": "蝾螈",
+  "entity.untamedwilds.newt_alpine": "高山螈",
+  "entity.untamedwilds.newt_alpine.sciname": "Ichthyosaura alpestris",
+  "entity.untamedwilds.newt_axolotl": "美西螈",
+  "entity.untamedwilds.newt_axolotl.sciname": "Ambystoma mexicanum",
+  "entity.untamedwilds.newt_coastal": "太平洋大鲵",
+  "entity.untamedwilds.newt_coastal.sciname": "Dicamptodon tenebrosus",
+  "entity.untamedwilds.newt_crested": "冠北螈",
+  "entity.untamedwilds.newt_crested.sciname": "Triturus cristatus",
+  "entity.untamedwilds.newt_eastern": "绿红东美螈",
+  "entity.untamedwilds.newt_eastern.sciname": "Notophthalmus viridescens",
+  "entity.untamedwilds.newt_fire_belly": "东方蝾螈",
+  "entity.untamedwilds.newt_fire_belly.sciname": "Cynops orientalis",
+  "entity.untamedwilds.newt_fire_salamander": "火蝾螈",
+  "entity.untamedwilds.newt_fire_salamander.sciname": "Salamandra salamandra",
+  "entity.untamedwilds.newt_marbled": "理纹欧螈",
+  "entity.untamedwilds.newt_marbled.sciname": "Triturus marmoratus",
+  "entity.untamedwilds.newt_ribbed": "西班牙肋突螈",
+  "entity.untamedwilds.newt_ribbed.sciname": "Pleurodeles waltl",
+  "entity.untamedwilds.newt_tiger": "虎纹蝾螈",
+  "entity.untamedwilds.newt_tiger.sciname": "Ambystoma tigrinum",
+  "item.untamedwilds.bucket_newt_coastal": "桶装太平洋大鲵",
+  "item.untamedwilds.bucket_newt_crested": "桶装冠北螈",
+  "item.untamedwilds.bucket_newt_eastern": "桶装绿红东美螈",
+  "item.untamedwilds.bucket_newt_fire_salamander": "桶装火蝾螈",
+  "item.untamedwilds.bucket_newt_ribbed": "桶装西班牙肋突螈",
+  "item.untamedwilds.bucket_newt_tiger": "桶装虎纹蝾螈",
+
+  "block.untamedwilds.anemone_rose_bulb": "红奶嘴海葵",
+  "block.untamedwilds.anemone_sand": "沙海葵",
+  "block.untamedwilds.anemone_sebae": "紫点海葵",
+  "block.untamedwilds.flora_orchid_magenta": "品红色树兰花",
+  "block.untamedwilds.flora_orchid_pink": "粉红色树兰花",
+  "block.untamedwilds.flora_orchid_purple": "紫色树兰花",
+  "block.untamedwilds.flora_orchid_red": "绯红色树兰花",
+  "block.untamedwilds.flora_common_reed": "普通芦苇",
+  "block.untamedwilds.flora_amazon_sword": "刺果泽泻",
+  "block.untamedwilds.flora_eelgrass": "鳗草",
+  "block.untamedwilds.flora_bush_temperate": "温带灌木丛",
+  "block.untamedwilds.flora_bush_creosote": "三齿拉雷亚灌木",
+  "block.untamedwilds.flora_elephant_ear": "象耳",
+  "block.untamedwilds.flora_hemlock": "毒参",
+  "item.untamedwilds.food_hemlock_stew": "谜之炖菜",
+  "block.untamedwilds.flora_titan_arum": "泰坦魔芋",
+  "item.untamedwilds.flora_titan_arum_corm": "泰坦魔芋球茎",
+  "block.untamedwilds.flora_zimbabwe_aloe": "津巴布韦芦荟",

```

```suggestion
  "block.untamedwilds.flora_zimbabwe_aloe": "高芦荟",
```
http://www.iplant.cn/info/Aloe%20excelsa